man/gemfile.5.ronn

```diff
@@ -504,3 +504,13 @@ bundler uses the following priority order:
      `rubygems.org`
   3. The sources specified via global `source` lines, searching each source in
      your `Gemfile` from last added to first added.
+
+## Should I check my `Gemfile.lock` into version control?
```

Comment:
This feels like this is missing a bit of context. We go from talking about the Gemfile.lock in your application but then jump into the Gemfile.lock inside a gem which is not imminently obvious.

How about we change the title to something like:

> Should I check my `Gemfile.lock` into version control inside my Gem?

?

Reply:
or maybe `for my gem?`

Reply:
ping @arbonap